lambda_functions/user_flows/steps.py

```diff
@@ -1,155 +1,173 @@
+def send_email_with_va_profile_id(notification_url: str, service_jwt: str, template_id: str) -> Response:
     payload = json.dumps({
         "template_id": template_id,
         "recipient_identifier": {
             "id_type": "VAPROFILEID",
-            "id_value": "1243"
+            "id_value": "1243",
         },
         "personalisation": {
             "claim_id": "600191990",
             "date_submitted": "October 30, 2020",
             "full_name": "VA Profile ID Email",
-            "first_name": "VAProfilefirstname"
-        }
+            "first_name": "VAProfilefirstname",
+        },
     })
+
     return send_email(notification_url, service_jwt, payload)
 
 
-def send_email_with_icn(notification_url: str, service_jwt: bytes, template_id: str) -> Response:
+def send_email_with_icn(notification_url: str, service_jwt: str, template_id: str) -> Response:
     payload = json.dumps({
         "template_id": template_id,
         "recipient_identifier": {
             "id_type": "ICN",
-            "id_value": "1008794780V325793"
+            "id_value": "1008794780V325793",
         },
         "personalisation": {
             "claim_id": "600191990",
             "date_submitted": "October 30, 2020",
             "full_name": "ICN Email",
-            "first_name": "ICNfirstname"
-        }
+            "first_name": "ICNfirstname",
+        },
     })
+
     return send_email(notification_url, service_jwt, payload)
 
 
 def get_notification_id(notification_response: Response) -> str:
     return notification_response.json()['id']
 
 
-def get_notification_status(notification_id: str, notification_url: str, service_jwt: bytes) -> Response:
-    return get_authenticated_request(F"{notification_url}/v2/notifications/{notification_id}", service_jwt)
+def get_notification_status(notification_id: str, notification_url: str, service_jwt: str) -> Response:
+    return get_authenticated_request(f"{notification_url}/v2/notifications/{notification_id}", service_jwt)
 
 
-def send_sms_with_phone_number(notification_url: str, service_jwt: bytes, template_id: str, recipient_number: str) -> Response:
+def send_sms_with_phone_number(notification_url: str, service_jwt: str, template_id: str, recipient_number: str) -> Response:
     payload = json.dumps({
         "phone_number": recipient_number,
-        "template_id": template_id
+        "template_id": template_id,
     })
 
     return send_sms(notification_url, service_jwt, payload)
 
 
-def send_sms_with_va_profile_id(notification_url: str, service_jwt: bytes, template_id: str) -> Response:
+def send_sms_with_va_profile_id(notification_url: str, service_jwt: str, template_id: str) -> Response:
     payload = json.dumps({
         "template_id": template_id,
         "recipient_identifier": {
             "id_type": "VAPROFILEID",
-            "id_value": "203"
-        }
+            "id_value": "203",
+        },
```

Comment:
Are you just adding commas here because it's proper JSON format?

Reply:
Makes sense. I didn't realize that was a part of good practices. Thanks! I'll have to include that in my future code.